tests/test_tools.py

```diff
@@ -973,21 +941,20 @@ def test_collect_appts(people, key, value, start, end, expected):
          {'date': '2019-09-06', 'postdoc_days': 30.5, 'ss_days': 25.5, 'student_days': 30.5},
          {'date': '2019-09-07', 'postdoc_days': 30.5, 'ss_days': 25.5, 'student_days': 30.5}]
          ),
-        ({'_id': 'malicious_grant', 'alias': 'very_malicious_grant'}, grant_people, '2012-12-23', '2013-01-24',
+        ({'_id': 'malicious_grant', 'alias': 'very_malicious_grant'}, appts, '2012-12-23', '2013-01-24',
```

Comment:
its a bit picky, but I don't think anyone in our group is being malicious.  Do you mean misbehaving_grant? or if we want more humour in our code (we do!) maybe "mischievous"

Reply:
yes sorry! the grant and person are now `magical` rather than `malicious`